cpp/tests/merge/merge_string_test.cpp

```diff
@@ -411,3 +411,48 @@ TYPED_TEST(MergeStringTest, Merge2StringKeyNullColumns)
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected_column_view2, output_column_view2);
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(expected_column_view3, output_column_view3);
 }
+
+class MergeLargeStringsTest : public cudf::test::BaseFixture {};
+
+TEST_F(MergeLargeStringsTest, MergeLargeStrings)
+{
```

Comment:
Super small nit:  maybe a negative test verifying 32 bit for small output columns?

Reply:
Wouldn't that be all the other tests in this file? 
Or do you mean adding a check to one of those tests that the offsets are 32-bit?

Reply:
> Wouldn't that be all the other tests in this file? Or do you mean adding a check to one of those tests that the offsets are 32-bit?

Right.  Or just an extension in this one that uses a smaller multiplier to produce a < 2 GB chars column.